web/viewer.css

```diff
@@ -291,7 +295,7 @@ html[dir='rtl'] #toolbarSidebar {
 
 #toolbarContainer, .findbar, .secondaryToolbar {
   position: relative;
-  height: 32px;
+  height: auto;
```

Comment:
I think that you should revert this change, since it has negative consequences for e.g. the display of the `loadingBar` (and possibly other things as well, that's just what I've seen so far).